public/app/features/plugins/admin/components/GetStartedWithPlugin/GetStartedWithDataSource.tsx

```diff
@@ -29,7 +31,15 @@ export function GetStartedWithDataSource({ plugin }: Props): React.ReactElement
   }
 
   return (
-    <Button variant="primary" onClick={onAddDataSource}>
+    <Button
+      variant="primary"
+      onClick={onAddDataSource}
+      disabled={
```

Comment:
Can we add a tooltip explaining to the user why they can't add a new datasource?

Reply:
I've updated the PR description with the tooltip screenshot. Let me know your thoughts